src/main/scala/gpp/ui/forms/EnumViewOptionalSelect.scala

```diff
@@ -0,0 +1,125 @@
+  minCharacters:        js.UndefOr[JsNumber] = js.undefined,
+  noResultsMessage:     js.UndefOr[ShorthandS[VdomNode]] = js.undefined,
+  onBlur:               js.UndefOr[Callback] = js.undefined,
+  onBlurE:              js.UndefOr[OnBlur] = js.undefined,
+  onClick:              js.UndefOr[Callback] = js.undefined,
+  onClickE:             js.UndefOr[OnClick] = js.undefined,
+  onChange:             js.UndefOr[FormDropdown.OnChange] = js.undefined,
+  onChangeE:            js.UndefOr[FormDropdown.OnChangeE] = js.undefined,
+  onClose:              js.UndefOr[Callback] = js.undefined,
+  onCloseE:             js.UndefOr[OnClose] = js.undefined,
+  onFocus:              js.UndefOr[Callback] = js.undefined,
+  onFocusE:             js.UndefOr[OnFocus] = js.undefined,
+  onLabelClick:         js.UndefOr[Callback] = js.undefined,
+  onLabelClickE:        js.UndefOr[OnLabelClick] = js.undefined,
+  onMouseDown:          js.UndefOr[Callback] = js.undefined,
+  onMouseDownE:         js.UndefOr[OnMouseDown] = js.undefined,
+  onOpen:               js.UndefOr[Callback] = js.undefined,
+  onOpenE:              js.UndefOr[OnOpen] = js.undefined,
+  onSearchChange:       js.UndefOr[OnSearchChange] = js.undefined,
+  onSearchChangeE:      js.UndefOr[OnSearchChangeE] = js.undefined,
+  open:                 js.UndefOr[Boolean] = js.undefined,
+  openOnFocus:          js.UndefOr[Boolean] = js.undefined,
+  placeholder:          js.UndefOr[String] = js.undefined,
+  pointing:             js.UndefOr[Pointing] = js.undefined,
+  renderLabel:          js.UndefOr[RenderLabel] = js.undefined,
+  required:             js.UndefOr[Boolean] = js.undefined,
+  scrolling:            js.UndefOr[Boolean] = js.undefined,
+  search:               js.UndefOr[Boolean | SearchFunction] = js.undefined,
+  searchInput:          js.UndefOr[ShorthandS[VdomNode]] = js.undefined,
+  searchQuery:          js.UndefOr[String] = js.undefined,
+  selectOnBlur:         js.UndefOr[Boolean] = js.undefined,
+  selectOnNavigation:   js.UndefOr[Boolean] = js.undefined,
+  selectedLabel:        js.UndefOr[JsNumber | String] = js.undefined,
+  simple:               js.UndefOr[Boolean] = js.undefined,
+  tabIndex:             js.UndefOr[String | JsNumber] = js.undefined,
+  text:                 js.UndefOr[String] = js.undefined,
+  tpe:                  js.UndefOr[String] = js.undefined,
+  trigger:              js.UndefOr[VdomNode] = js.undefined,
+  upward:               js.UndefOr[Boolean] = js.undefined,
+  width:                js.UndefOr[SemanticWidth] = js.undefined,
+  wrapSelection:        js.UndefOr[Boolean] = js.undefined,
+  modifiers:            Seq[TagMod] = Seq.empty
+)(implicit
+  val enum:             Enumerated[A],
+  val display:          Display[A],
+  val effect:           Effect[F]
+) extends ReactProps[EnumViewSelectBase](EnumViewSelectBase.component)
+    with EnumViewSelectBase {
+
+  type AA    = A
+  type BB    = Option[A]
+  type FF[X] = F[X]
+
+  val multiple = false
```

Comment:
Is this being used? or is it an override?

Reply:
> Is this being used? or is it an override?

That's an override.

Reply:
Maybe we should mark them as override or it could be easily deleted

Reply:
Will do. A bad habit from my last contract, where I was constantly being told to remove the override keyword in PRs. 😆